easybuild/scripts/bootstrap_eb.py

```diff
@@ -519,6 +532,8 @@ def main():
 # EasyBuild is a (set of) Python packages, so it depends on Python
 # usually, we want to use the system Python, so no actual Python dependency is listed
 allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+preinstallopts = '%(preinstallopts)s'
```

Comment:
i was talking about the easyconfig for easybuild itself. i don't see why the bootrap easyconfig for EB is different from the one in the repos (i also don't see why don't use the one from the repos)